### PR TITLE
dcache-common: refactoring slf4j logging messages and logger variable name

### PR DESCRIPTION
**Pull request by marisanest**
Motivation: with normal string concatenations in log-messages strings are always build, regardless if log-level is activated or not. with parameterized log-messages the strings only become build, when the log-level is activated. With a consistent variable name for the logger the application gets more standardized.

Modification: using placeholder with parameterized messages instead of string concatenations. Naming all logger variables "logger".

Result: gained efficiency and standardization

Signed-off-by: marisanest <marisanest@mailbox.org>

### CODE REVIEW

File: modules/common/src/main/java/org/dcache/auth/OpenIdCredentialRefreshable.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 public class OpenIdCredentialRefreshable extends WrappingOpenIdCredential
 {
-    private static final Logger LOG =
+    private static final Logger logger =
```
Comment (marisanest) [inline modules/common/src/main/java/org/dcache/auth/OpenIdCredentialRefreshable.java]: Hi, thanks for your feedback! We (lotta and I) saw different naming styles within the code. One was "LOG", one "_log" and the last one "logger". We thought that it might be good to unify these names. After some researches we thought that "logger" would be the best name because a logger reference is not a constant value and therefore uppercase is maybe not the best case :) But if you already have a certain naming convention it might be superfluous to change that. Can I make the changes directly in this pull request or should I make a new one without renaming?

File: modules/common/src/main/java/org/dcache/auth/OpenIdCredentialRefreshable.java [hunk 2]
```diff
@@ -42,7 +42,7 @@ public String getBearerToken()
             try {
                 refreshOpenIdCredentials();
             } catch (IOException | AuthenticationException e) {
-                LOG.warn("Error Refreshing OpenId Bearer Token with {}: {}",
+                logger.warn("Error Refreshing OpenId Bearer Token with {}: {}",
```
Comment (marisanest) [inline modules/common/src/main/java/org/dcache/auth/OpenIdCredentialRefreshable.java]: Hi, thanks for the tip! We will consider it in our future requests :)

Comment (kofemann) [inline modules/common/src/main/java/org/dcache/auth/OpenIdCredentialRefreshable.java]: Ok :)

File: modules/common/src/main/java/org/dcache/auth/OpenIdCredentialRefreshable.java (hunk 1 above)
Comment (kofemann) [inline modules/common/src/main/java/org/dcache/auth/OpenIdCredentialRefreshable.java]: That's the point. You see, that we are already confused, so don't make it worse :). If you want to enforce a single style, then you can do it with a dedicated commit and describe there why.

**Comment by kofemann** [comment]
ok to test

**Comment by kofemann** [comment]
Thanks! Manually merged as eb9b04f981
